encoding/m3tsz/roundtrip_test.go

```diff
@@ -220,3 +225,20 @@ func generateMixedDatapoints(numPoints int, timeUnit time.Duration) []ts.Datapoi
 	}
 	return res
 }
+
+func generateOverflowDatapoints() []ts.Datapoint {
+	var startTime int64 = 1427162462
+	currentTime := time.Unix(startTime, 0)
+	largeInt := float64(math.MaxInt64 - 1)
```

Comment:
Shouldn't this be `float64(math.MaxInt64)-1` to be more safe about compiler semantics here?  Same with the variable below.